### PR TITLE
Migration verbiage changes

### PR DESCRIPTION
**Pull request by develohpanda**
image

image

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/wrapper-migration.js [hunk 1]
```diff
@@ -77,11 +77,15 @@ const Options = ({ start, cancel }: OptionsProps) => {
   return (
     <>
       <p>
-        <strong>Would you like to migrate data from Insomnia Designer?</strong>
+        <strong>Migrate from Insomnia Designer</strong>
+      </p>
+      <p>
+        Insomnia Designer and Core are now Insomnia! Select the items below you'd like to migrate
+        from Designer.
```
Comment (develohpanda) [inline packages/insomnia-app/app/ui/components/wrapper-migration.js:84]: This one seems interesting; a lot of things will auto migrate but this wording implies they can only choose the three things (settings, plugins, responses).

Comment (reynolek) [reply packages/insomnia-app/app/ui/components/wrapper-migration.js:84]: Are the things that would auto migrate directly associated with the 3 listed things?

Comment (develohpanda) [reply packages/insomnia-app/app/ui/components/wrapper-migration.js:84]: Nope; the additional items are all the database models (which users will recognize as workspaces, requests, environments), so naturally they're not optional. These settings are just some additional options they have.

Comment (reynolek) [reply packages/insomnia-app/app/ui/components/wrapper-migration.js:84]: Perhaps we can do something like "Along with the required items, you may optionally migrate the below items from Designer" or something? @nijikokun

Comment (nijikokun) [reply packages/insomnia-app/app/ui/components/wrapper-migration.js:84]: Actually, thinking, we should let users have the choice of whether or not to migrate workspaces (disabling should disable responses as well)

Comment (develohpanda) [reply packages/insomnia-app/app/ui/components/wrapper-migration.js:84]: INS-469

**Review by nijikokun — APPROVED** [review]

**Pull request merged**